.buildkite/postgres-config.yaml

```diff
@@ -3,7 +3,7 @@
 # CI's Docker setup at the point where this file is considered.
 server_name: "localhost:8800"
 
-signing_key_path: "/src/.buildkite/test.signing.key"
+signing_key_path: ".buildkite/test.signing.key"
```

Comment:
the `/src` was redundant, inconsistent with the relative `database: ".buildkite/test_db.db"` in `sqlite-config.yaml`, and made it fiddly to test outside docker.